omise/css/omise_internet_banking.css

```diff
@@ -27,11 +27,7 @@
   width: 30px;
   height: 30px;
 }
-.omise-internet-banking .omise-bank-text-wrapper { display: inline-block; }
-.omise-internet-banking .secondary-text {
-  font-size: 80%;
-  color: #aaa;
-}
+.omise-internet-banking .omise-bank-text-wrapper { display: inline-block; line-height:38px; }
```

Comment:
@jonrandy would it better to set `line-height` inside `span.title`? 
something like 👇 
```css
.omise-bank-text-wrapper .title { line-height:38px; }
```

Just thought that `display: inline-block` is unnecessary here (maybe)

Reply:
I reworked the html a bit now, there was some `div`-itis going on there. Less HTML and CSS now, same result